scripts/upgrade

```diff
@@ -239,7 +239,7 @@ ynh_app_setting_set --app=$app --key=multisite --value=$multisite
 ynh_script_progression --message="Updating plugins" --weight=11
 
 # wget -nv https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar -O $final_path/wp-cli.phar
-# wpcli_alias="php $final_path/wp-cli.phar --allow-root --path=$final_path"
+# wpcli_alias="php7.0 $final_path/wp-cli.phar --allow-root --path=$final_path"
```

Comment:
Why is this line commented?

Reply:
I wondered the same thing yesterday...
[...]
Ok, because we currently have a temporary fix before in the script that already does that
https://github.com/YunoHost-Apps/wordpress_ynh/blob/59baee2ef9d85e3284ecf47fc3c7bd16a3c08ac3/scripts/upgrade#L132-L136

Reply:
I guess we can remove it then?

Reply:
I would rather keep it here, since it's its original place.
The previous part is not suppose to stay forever. Just a migration code.

Reply:
ok, not a problem anyway